### PR TITLE
Fixed incompatability in persistence for older versions

### PR DESCRIPTION
**Pull request by chinmayapancholi13**
Added `load` method to class `gensim.models.wrappers.fasttext.FastText` to set `self.wv.syn0_ngrams` (present in newer models) using `self.wv.syn0_all` (present in older models).

Relevant issue : https://github.com/RaRe-Technologies/gensim/issues/1642
cc : @Liebeck

### CODE REVIEW

**Comment by menshikh-iv** [comment]
Thanks @chinmayapancholi13, good work :+1:

File: gensim/models/wrappers/fasttext.py [hunk 1]
```diff
@@ -251,6 +251,14 @@ def load_fasttext_format(cls, model_file, encoding='utf8'):
         model.load_binary_data(encoding=encoding)
         return model
 
+    @classmethod
+    def load(cls, *args, **kwargs):
+        model = super(FastText, cls).load(*args, **kwargs)
+        if hasattr(model.wv, 'syn0_all'):
+            setattr(model.wv, 'syn0_ngrams', model.wv.syn0_all)
```
Comment (piskvorky) [inline gensim/models/wrappers/fasttext.py]: What's with this `setattr` / `delattr`? Why not a simple (and more readable) `model.wv.syn0_ngrams = model.wv.syn0_all`?

File: gensim/test/test_fasttext_wrapper.py [hunk 2]
```diff
@@ -351,6 +351,22 @@ def testConsistentDtype(self):
...
 
+    def testPersistenceForOldVersions(self):
+        """Test backward compatibility for models saved with versions < 3.0.0"""
+        old_model_path = datapath('ft_model_2.3.0')
+        loaded_model = fasttext.FastText.load(old_model_path)
+        self.assertEqual(loaded_model.vector_size, 10)
+        self.assertEqual(loaded_model.wv.syn0.shape[1], 10)
+        self.assertEqual(loaded_model.wv.syn0_ngrams.shape[1], 10)
+        # in-vocab word
+        in_expected_vec = numpy.array([-2.44566941, -1.54802394, -2.61103821, -1.88549316, 1.02860415,
```
Comment (piskvorky) [inline gensim/test/test_fasttext_wrapper.py]: Bad indentation (please use handing indent).

File: gensim/test/test_fasttext_wrapper.py [hunk 3]
```diff
@@ -351,6 +351,22 @@ def testConsistentDtype(self):
...
+            1.19031894, 2.01627707, 1.98942184, -1.39095843, -0.65036952])
+        self.assertTrue(numpy.allclose(loaded_model["the"], in_expected_vec, atol=1e-4))
+        # out-of-vocab word
+        out_expected_vec = numpy.array([-1.34948218, -0.8686831, -1.51483142, -1.0164026, 0.56272298,
```
Comment (piskvorky) [inline gensim/test/test_fasttext_wrapper.py]: dtto

**Pull request merged**